build_easily.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+log_dir=$TRAF_HOME/../../"build_logs"
+
+function process(){
+    component_name=$1
+    cmd=$2
+    log_file=$3
+    make_path=$4
+    printf "[%-30s]........................[\e[33m%s\e[0m]\r" "building $component_name" "waiting"
+    cd $make_path
+    #echo $log_file
+    ret=$($cmd > "$log_dir/$log_file" 2>&1;echo $?)
+#    ret=0
+    if [ 0 -eq $ret ]; then
+        printf "[%-30s]...................................[\e[32m%s\e[0m]\r\n" "build $component_name" "ok"
+    else
+        printf "[%-30s]...................................[\e[31m%s\e[0m]\r\n" "build $component_name" "failed"
+        printf "\e[31mBUILD FAILED\e[0m\n"
+        exit 1
+    fi
+
+}
+
+if [ ! -d $log_dir ];then
+    mkdir $log_dir
+fi
+
+if [ ! -d $log_dir/"foundation" ];then
+    mkdir $log_dir/"foundation"
+fi
+
+#export LC_ALL=en_US.UTF-8
+cd $TRAF_HOME/../
+./bldenvchk.sh;
+export SQ_BUILD_TYPE=debug
```

Comment:
you can use debug as default, and developer can change it via parameter.

Reply:
Hi, this srcipt now can olny be used in debug env.